addons/mail_group/views/mail_template_data.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" ?>
```

Comment:
as it conflicts with mail.template files holding data, maybe a "mail_template_email_layouts.xml" file, like in mail ? To put in data ?